zubhub_frontend/zubhub/public/locales/en/translation.json

```diff
@@ -756,5 +758,148 @@
+      },
+      "materialsUsed": {
+      "label": "what materials did you use?",
+      "helperText": "List some of the essential materials used to create this activity",
+      "encouragingText": "We strongly encourage you to upload an image containing all materials and tools used to create this activity",
+      "images": {
+        "label": "Add Images"
+      },
+      "errors": {
+        "max": "your material shouldn't be more than 50 characters",
+        "required": "this field is required",
+        "required1": "At least one material is required!",
+        "min": "your material should be at least 2 characters"
+      }
+      },
+      "facilitationTips": {
+        "label": "Facilitation tips",
+        "helperText": "List some helpful facilitation tips",
+        "placeholder": "Facilitation tips",
+        "errors": {
+          "max": "your facilitation tips shouldn't be more than 10,000 characters",
+          "required": "This field is required"
+        }
+
+      },
+      "activityImages": {
+        "label": "Add Images of your activity",
+        "errors": {
+          "required1": "you can't create a new activity without images",
+          "required": "This field is required",
+          "onlyImages": "Only images are allowed for this field",
+          "imageOrVideo": "You must provide either image(s), video file, or video url to create your activity!",
+          "imageSizeTooLarge": "One or more of your images is larger than 10 MB in size! Upload smaller images",
+          "tooManyImages": "Too many images uploaded! We only support 10 images per activity :("
+        }
+      },
+      "making_steps": {
+        "label": "Creation Steps",
+        "helperText": "List steps you followed to create this activity",
+        "placeholder": "creation step",
+        "addMore": "Add a Step",
+        "image": {
+          "label": "Add Step Image"
+        },
+        "errors": {
+          "max": "your creation step shouldn't be more than 10,000 characters",
+          "required": "describe creation process in at least one step ",
+          "emptyStep": "step can't be empty provide either a description or an image"
+        }
+      },
+      "inspiringArtist": {
+        "label": "Inspiring Artist",
+        "helperText": "Enter a Bio of your inspiring artist",
+        "placeholder": "Inspiring artist Bio",
+        "image": {
+          "label": "Add Artist Image"
+        },
+        "errors": {
+          "max": "your artist bio shouldn't be more than 10,000 characters",
+          "required": "This field is required"
+        }
+      },
+      "inspiringExemples": {
+        "label": "inspiring Examples",
+        "addMore": "Add Example",
+        "image": {
+          "label": "Add Example Image"
+        }
+      },
+      "image": "Image added",
+      "images": "Images added",
+      "videoFile": "Video file added",
+      "videoURL": "Video URL added"  
+    },
+    "buttons": {
+      "Next": "Next->",
```

Comment:
Use the same icons we are using for "previous and next page buttons" on the homepage